src/main/java/com/example/bookstore/controller/BookController.java

```diff
@@ -0,0 +1,34 @@
+package com.example.bookstore.controller;
+
+import com.example.bookstore.dto.BookDto;
+import com.example.bookstore.dto.CreateBookRequestDto;
+import com.example.bookstore.service.BookService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController()
```

Comment:
```suggestion
@RestController
@RequiredArgsConstructor
@RequestMapping("/api/books")
```

and you can delete @Autowired annotation and some part from @GetMapping